helm/cluster-vsphere/templates/kubeadmcontrolplane.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+  annotations:
+    {{- include "helm-keep" . | nindent 4 }}
```

Comment:
Just curious: what's the benefit of including a defined template over just using `"helm.sh/resource-policy": "keep"` here?

Reply:
right, this is simpler. My thinking was probably "make it DRY" so that it can be easily changed all at one place, but they the name of that template should be something like "resource-policy" and not "helm-keep"